src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -7511,6 +7511,118 @@ void CodeGen::genArm64EmitterUnitTestsSve()
                               INS_SCALABLE_OPTS_UNPREDICATED);
     theEmitter->emitIns_R_R_I(INS_sve_str, EA_SCALABLE, REG_V2, REG_R3, 255, INS_OPTS_NONE,
                               INS_SCALABLE_OPTS_UNPREDICATED);
+
+    // IF_SVE_HY_3A
+    theEmitter->emitIns_PRFOP_R_R_R(INS_sve_prfb, EA_SCALABLE, SVE_PRFOP_PLDL1KEEP, REG_P1, REG_R2, REG_V3,
```

Comment:
When I was doing `ptrue`, that uses an enum. I considered squashing the enum into `INS_SCALABLE_OPTS`. But, here we require explicit sopt values. Therefore it's best to keep as is

tldr: all good :)

Reply:
Yea, I considered `INS_SCALABLE_OPTS`, but it would explode in the number of cases since some of these instructions need to take UXTX and SXTX.